distribution/benchmark/runclass.sh

```diff
@@ -24,13 +24,39 @@ fi
 BASE_DIR=$(dirname $0)/..
 CLASSPATH=.:${BASE_DIR}/conf:${CLASSPATH}
 
+choose_gc_log_directory()
+{
+    case "`uname`" in
+        Darwin)
+            if [ ! -d "/Volumes/RAMDisk" ]; then
+                # create ram disk on Darwin systems as gc-log directory, default size is 600MB
+                DIR_SIZE_IN_MB=600
```

Comment:
This size could be a variable.

Reply:
Do you mean that I should put it in a config file? or move it outside the choose_gc_log_directory function as a parameter?

Reply:
The latter, the first step.

Reply:
OK, modified.